scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -91,7 +91,9 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
       {% endif %}
 
       <div class="row ">
-        <div class="col-xs-12 col-md-6">{{ causatives_list(causatives, partial_causatives, institute, case) }}</div>
+        {% if not case.track == 'cancer' %}
+          <div class="col-xs-12 col-md-6">{{ causatives_list(causatives, partial_causatives, institute, case) }}</div>
```

Comment:
We will need to differentiate further to remove this methinks? At least for inherited cancer and hematology it makes a lot of sense to work with "causative" variants. If you worry about the wording for a complex solid tumor, perhaps we could introduce concepts like "driver, putative" or qualify the case highlighted variants with "actionable", "biomarker" - or make it "phenotypically significant" variants?